lib/buzz_os.zig

```diff
@@ -494,7 +494,7 @@ export fn SocketServerStart(vm: *api.VM) c_int {
 
     server.listen(list.addrs[0]) catch |err| {
         switch (err) {
-            error.NoDevice => vm.bz_pushErrorEnum("lib.errors.FileSystemError", "lib.errors.FileSystemError".len, "NoDevice", "NoDevice".len),
```

Comment:
did you run this against zig master? this is why I added this case but they maybe reverted this on later commits?

Reply:
you can see if failed in the github action

Reply:
with version 0.10.0-dev.4247+3234e8de3